ledger/Cargo.toml

```diff
@@ -0,0 +1,45 @@
+[package]
+name = "filecoin-signer-ledger"
+description = "Library to integrate with the Ledger Filecoin app"
+version = "0.10.0"
+license = "Apache-2.0"
+authors = ["Zondax GmbH <info@zondax.ch>"]
+homepage = "https://github.com/ZondaX/ledger-filecoin-rs"
+repository = "https://github.com/ZondaX/ledger-filecoin-rs"
+readme = "README.md"
+categories  = ["authentication", "cryptography"]
+keywords = ["ledger", "nano", "apdu", "filecoin"]
+edition     = "2018"
+autobenches = false
+
+[lib]
+name = "filecoin_signer_ledger"
+
+[dependencies]
+byteorder = "1.3.1"
+matches = "0.1.8"
+lazy_static = "1.2.0"
+libsecp256k1 = "0.3.5"
+blake2b_simd = "0.5.10"
+futures = "0.3.4"
+serde = { version = "1.0", features = ["derive"] }
+thiserror = "1.0.16"
+
+# Used for internal development
+#ledger-transport = { path = "../../ledger-rs/ledger-transport"}
+#bip44 = { path = "../../ledger-rs/bip44"}
+ledger-transport = { git = "https://github.com/Zondax/ledger-rs", branch = "feature/refactoring-wasm"}
```

Comment:
Let's open an issue to remember that we need to change this once we merge the other PR